packages/vite/src/node/preview.ts

```diff
@@ -55,8 +55,7 @@ export interface PreviewServer {
 
 /**
  * Starts the Vite server in preview mode, to simulate a production deployment
- * @param config - the resolved Vite config
- * @param serverOptions - what host and port to use
+ * @param inlineConfig - the resolved Vite config
```

Comment:
It isn't the resolved config, it is a UserConfig with additional properties. There isn't a JSDoc param in other places, we can remove it here directly
```suggestion
```